app/concepts/ticket/use_case/taken.rb

```diff
@@ -0,0 +1,17 @@
+module UseCase
+    module Tickets
+      class Taken < UseCase::Base::Base
+
+        def call
+          seats_taken=[]
+          i = 0
+          repository.all.each do |ticket|
+            seats_taken[i]=ticket[:seat]
+            i=i+1
+          end
+          return seats_taken
```

Comment:
More ruby way:

```ruby
def call
  repository.all.each_with_index.each_with_object({}) { |(ticket, index), result| result[index] = ticket[:seat] }
end
```

or

```ruby
def call
  repository.all.each_with_index.map { |ticket, index| [index, ticket[:seat]] }.to_h
end
```

Reply:
@jandudulski Ok, so there is another solution for that, gonna change it, way clearer.